### PR TITLE
Add checksum for numba 0.55.2 and 0.56, py-llvmlite 0.39.0

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Review by tldahlgren — DISMISSED** [review]
Confirmed sha256.

**Comment by tldahlgren** [comment]
Will defer merge decision to give @adamjstewart a chance to comment.

File: var/spack/repos/builtin/packages/py-numba/package.py [hunk 1]
```diff
@@ -28,14 +30,16 @@ class PyNumba(PythonPackage):
     depends_on('python@3.6:3.8', type=('build', 'run'), when='@0.48:0.51')
     depends_on('python@3.3:3.7', type=('build', 'run'), when='@0.40.1:0.47')
     depends_on('python@3.3:3.6', type=('build', 'run'), when='@:0.35.0')
-    depends_on('py-numpy@1.18:1.21', type=('build', 'run'), when='@0.55.0:')
+    depends_on('py-numpy@1.18:1.22', type=('build', 'run'), when='@0.55.0:')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-numba/package.py]:
```suggestion
    depends_on('py-numpy@1.18:1.22', type=('build', 'run'), when='@0.55.2:')
```

File: var/spack/repos/builtin/packages/py-numba/package.py [hunk 2]
```diff
@@ -28,14 +30,16 @@ class PyNumba(PythonPackage):
...
+    depends_on('py-numpy@1.18:1.21', type=('build', 'run'), when='@0.55.0:0.55.1')
     depends_on('py-numpy@1.17:1.20', type=('build', 'run'), when='@0.54')
     # set upper bound for py-numpy the same as newer release
     depends_on('py-numpy@1.15:1.20', type=('build', 'run'), when='@0.48:0.53')
     depends_on('py-numpy@1.10:1.20', type=('build', 'run'), when='@:0.47')
     depends_on('py-setuptools', type=('build', 'run'))
 
-    depends_on('py-llvmlite@0.38', type=('build', 'run'), when='@0.55.1')
+    depends_on('py-llvmlite@0.39', type=('build', 'run'), when='@0.56.0')
+    depends_on('py-llvmlite@0.38', type=('build', 'run'), when='@0.55.1:0.55')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-numba/package.py]:
```suggestion
    depends_on('py-llvmlite@0.39', type=('build', 'run'), when='@0.56')
    depends_on('py-llvmlite@0.38', type=('build', 'run'), when='@0.55')
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**